tests/components/test_entity_recognizer.py

```diff
@@ -11,7 +11,7 @@ def test_memm_model(kwik_e_mart_app_path):
         "k": 5,
         "scoring": "accuracy",
         "grid": {
-            "penalty": ["l1", "l2"],
+            "penalty": ["l2"],
```

Comment:
Could remove this since it is the default.

Reply:
I tried removing it but it failed on the assert a few lines below so I'm keeping it for the time being.